styles/buttons-all.css

```diff
@@ -0,0 +1,91 @@
+.component-btn {
+  width: 80%;
+  height: 200px;
+  background: white;
+  justify-self: center;
+  border-radius: 15px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.primary {
+  padding: 1.2rem 2.5rem;
+  border-radius: 35px;
+  border: none;
+  font-size: 1.2rem;
+  background: rgb(7, 7, 65);
+  color: white;
+  cursor: pointer;
+}
+
```

Comment:
This code repetition can be avoided if you'll create ```.btn``` class and add to all other components as above mentioned.